tests/test_modeling.py

```diff
@@ -83,25 +90,25 @@ def test_vhead(self):
         for model_name in self.all_model_names:
             model = AutoModelForCausalLMWithValueHead.from_pretrained(model_name)
             self.assertTrue(hasattr(model, "v_head"))
-    
+
     def test_vhead_nb_classes(self):
```

Comment:
would this be better?
```suggestion
    def test_vhead_shape(self):
```

Reply:
I'd address this too in a separate PR.

Reply:
Addressed in https://github.com/lvwerra/trl/pull/65 !